### PR TITLE
Home empty state

### PR DESCRIPTION
**Pull request by ValesarroRS**
## Pull request type

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Inside pages used to be a place holder.

## What is the new behavior?

Implemented welcome page and home empty state page
Added target icon
Refactor on some components

## Preview

Screen Shot 2022-11-23 at 14 03 25
Screen Shot 2022-11-23 at 14 04 07

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -26,6 +27,7 @@ const App = () => {
                 }
               >
                 <Route path="/welcome" element={<Welcome />} />
+                <Route path="/homeEmptyState" element={<HomeEmptyState />} />
```
Comment (Alejoboga20) [inline src/App.js:30]: I'd recommend to create a constant to store your different routes and use it wherever you need routes

File: src/components/HomeEmptyState/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.footer {
+    height: 1.75rem;
+    width: 3rem;
+    margin-bottom: 1.438rem;
+}
+
+.containerHomeEmpty {
```
Comment (Alejoboga20) [inline src/components/HomeEmptyState/index.module.scss]: homeEmptyContainer makes more sense to me

File: src/components/HomeMenu/index.jsx [hunk 3]
```diff
@@ -1,15 +1,13 @@
...
-    <div className="HomeMenu">
-      <button>X</button>
-      <div id="Options">
-        <p>About</p>
-        <p>Contact</p>
-      </div>
+const HomeMenu = () => (
+  <div className="HomeMenu">
+    <button>X</button>
+    <div id="Options">
```
Comment (Alejoboga20) [inline src/components/HomeMenu/index.jsx]: WDYT about using a class instead of id?

File: src/components/shared/Button/index.module.scss [hunk 4]
```diff
@@ -31,7 +31,16 @@ button {
 .smallLinkText {
     font-family: 'Open Sans', sans-serif;
     font-size: 0.688rem;
-    margin-top: 0.938rem;
     text-transform: none;
     letter-spacing: normal;
 }
+
+.mediumLinkText {
+    @extend .smallLinkText;
```
Comment (Alejoboga20) [inline src/components/shared/Button/index.module.scss:39]: you could create a class called` linkText` and extend that one to small, medium, etc

File: src/components/shared/Profile/index.jsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+import SplitBar from "components/shared/SplitBar";
+import Description from "components/shared/Description";
+
+const Profile = () => {
+  const dispatch = useDispatch();
+
+  const [logOutRequest] = useDeleteLogOutMutation();
+
+  async function logOut() {
+    try {
```
Comment (Alejoboga20) [inline src/components/shared/Profile/index.jsx:18]: you can use redux matchers to avoid using try/catch here.

**Review by Alejoboga20 — APPROVED** [review]
Good job. Just a few comments as suggestion

File: src/components/HomeEmptyState/index.module.scss (hunk 2 above)
Comment (ValesarroRS) [reply src/components/HomeEmptyState/index.module.scss]: Fixed!

File: src/components/HomeMenu/index.jsx (hunk 3 above)
Comment (ValesarroRS) [reply src/components/HomeMenu/index.jsx]: Fixed!

File: src/App.js (hunk 1 above)
Comment (ValesarroRS) [reply src/App.js:30]: Will do this and other improvements on a separate PR, thanks for the suggestion!

File: src/components/shared/Button/index.module.scss (hunk 4 above)
Comment (ValesarroRS) [reply src/components/shared/Button/index.module.scss:39]: Done!

File: src/components/shared/Profile/index.jsx (hunk 5 above)
Comment (ValesarroRS) [reply src/components/shared/Profile/index.jsx:18]: Will investigate more about redux matchers to implement it on the next PR. Thanks for the suggestion!

**Pull request merged**